### ISSUE TITLE
Werthles Headless Kit not implemented

### ISSUE THREAD

**Issue by Fourjays** [blackwatchint/blackwatchint#166]
Headless clients were not functioning during the op. After some investigation it appears that the script has not been implemented as previously thought.

**Comment by Fourjays** [comment]
Need to include the WerthlesHeadless.sqf file in the template structure somewhere and then call it using the following line in the init.sqf:
```sqf
[true,30,false,true,30,3,true,[]] execVM "WerthlesHeadless.sqf";
```

**Comment by ShiFoo** [comment]
This script should be put into "BWI\scripts". Keep in mind that this also changes the call:

[true,30,false,true,30,3,true,[]] execVM "BWI\scripts\WerthlesHeadless.sqf";

**Comment by Fourjays** [comment]
Fixed by @wingatex in 6a40aff2642446f917ea6d30055ce634bcd30ed9.
